app/models/chef.rb

```diff
@@ -1,4 +1,7 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+
```

Comment:
For US 3, in order to get the unique ingredients that each chef uses, an easier method to do this would be to:
1. Add an additional association to this model to create a direct association to ingredients. Since the chef already has many dishes, you can add `has_many :ingredients, through: :dishes` to create a shortcut directly to ingredients. We're able to do this because that chain already exists from chefs to dishes, and then from dishes to ingredients. Adding another association just lets us get there directly without writing more joins in AR.

2. Then, once we have this association, we can have a model method here that will just grab a distinct list of those ingredients:
```
def ingredients_list
     self.ingredients.distinct
end
```